### PR TITLE
[CSL-1758] Merge to master

### PR DESCRIPTION
**Pull request by int-index**

### CODE REVIEW

File: update/Pos/Update/Download.hs [hunk 1]
```diff
@@ -79,29 +85,39 @@ downloadUpdate cps = do
...
             cpsUpdateProposal
+    onAlreadyInstalled =
+        logInfo $
+        sformat
+            ("We won't download an update for proposal "%build%
+             ", because it's already installed")
+            proposalToDL
 
--- Download and save archive update by given `ConfirmedProposalState`
-downloadUpdateDo :: UpdateMode ctx m => ConfirmedProposalState -> m ()
```
Comment (gromakovsky) [inline update/Pos/Update/Download.hs]: Obsolete formatting

File: lib/src/Pos/DB/Misc.hs [hunk 2]
```diff
@@ -8,6 +8,7 @@ module Pos.DB.Misc
        , addProxySecretKey
        , removeProxySecretKey
        , dropOldProxySecretKeys
+
```
Comment (gromakovsky) [inline lib/src/Pos/DB/Misc.hs]: 🤔

**Comment by gromakovsky** [comment]
> [CSL-1856] Move update MiscDB to cardano-sl-update

Oh wait, this is CSL-1758, can you fix?

**Comment by int-index** [comment]
> Oh wait, this is CSL-1758, can you fix?

Oops! Fixed.

**Pull request merged**